x-pack/plugins/index_lifecycle_management/public/store/middleware/auto_set_node_attrs.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  setPhaseData, setSelectedNodeAttrs
+} from '../actions';
+import { getPhaseData } from '../selectors';
+import { PHASE_WARM, PHASE_NODE_ATTRS, PHASE_COLD } from '../constants';
+
+export const autoSetNodeAttrs = store => next => action => {
+  const state = store.getState();
+
+  if (action.type === setSelectedNodeAttrs().type) {
```

Comment:
Same comments apply to this file about creating actions only to check their type.

Reply:
constantized